src/internal/sdata/sdata_test.go

```diff
@@ -74,6 +80,21 @@ func TestMaterializeSQL(t *testing.T) {
 	}
 }
 
+// newSnowSQL creates an emphermeral database in a real Snowflake instance.
+func newSnowSQL(t testing.TB) *sqlx.DB {
+	user := os.Getenv("SNOW_USER")
+	password := os.Getenv("SNOW_PASSWORD")
+	account_identifier := os.Getenv("SNOW_ACCOUNT")
+	dsn := fmt.Sprintf("%s:%s@%s", user, password, account_identifier)
+
+	db, err := sqlx.Open("snowflake", dsn)
```

Comment:
We should use our own `pachsql.OpenURL` so that code path also gets exercised during this test.